### PR TITLE
Create Design Review Reminder 

### PR DESCRIPTION
**Pull request by toviet**

### CODE REVIEW

File: src/utils.ts [hunk 1]
```diff
@@ -315,10 +328,16 @@ const getEstimateForStoryType = (story: PivotalStory) => {
  * @param  {StoryResponse} story
  * @returns string
  */
-export const getPrDescription = (body: string = '', story: PivotalStory): string => {
+export const getPrDescription = (body: string = '', story: PivotalStory, reviews: PivotalStoryReview[]): string => {
   const { url, id, story_type, labels, description, name } = story;
   const labelsArr = (labels as Label[]).map((label: { name: string }) => label.name).join(', ');
 
+  // Mention Me Design Review Type IDs
+  const mmDesignReviewTypes = [4013928, 4884434];
```
Comment (edhgoose) [inline src/utils.ts]: I'd be very careful here.

This is a fork of a public repo - and generally, it's best, where possible to always make changes which can be merged upstream so others can benefit.

This is highly specific to Mention Me. This means our fork is going to be permanent - and we probably don't really want to do that. It'll make it hard to bring in changes from upstream (and we don't get to contribute it back this way either).

Can we think of a better way to do this? Could we make it generic? (See comments further down too - you might have painted yourself into a corner which I can help you out of!)

File: src/utils.ts [hunk 2]
```diff
@@ -353,6 +372,15 @@ export const getPrDescription = (body: string = '', story: PivotalStory): string
   </table>
 </details>
 <br />
+
+${requireDesignReview && `<details open>
+  <summary><strong>Pivotal Design Review</strong></summary>
+  <br />
+  
+  - [ ] Design review is required for this PR, make sure to ask @AMS-00 for review once finished.
```
Comment (edhgoose) [inline src/utils.ts]: There's a few things in here that I think might be worth considering.

1. Does Anna really want a notification every time you add a review, and what happens when she hires a new colleague?
2. How might we want to extend this in the future - imagine we had a security team and wanted a security review for instance? Or we wanted Anca and Carly to have a review?
3. How could a non Mention Me user use this feature?
4. I'm guessing the review objects are specific to the projects we have. What happens when we add more projects?

As a clue, you may be better off creating something that looks like this:

```
# Reviews required
- [ ] Design (Anna Stark)
- [ ] Security (Tony Stark)
- [ ] Product (Anca Filip)
```

**Review by edhgoose — CHANGES_REQUESTED** [review]
Looks good in principle @toviet - but I'd recommend going a bit further to make it really valuable. Have a look at the comments and see what you think.

Tiny piece of feedback: do try and use branch names which are descriptive in the future. I know you had some permission problems, but it helps people to follow what is it and why it matters.

File: src/utils.ts (hunk 2 above)
Comment (toviet) [reply src/utils.ts]: I really did not want to use the explicit IDs to compare, however couldn't find any API end point to extract the reviewer or review type name based on the Ids. I will dig some more

Comment (toviet) [reply src/utils.ts]: Found it now :)

**Pull request closed without merging**